src/ApparentHorizons/ComputeExcisionBoundaryVolumeQuantities.hpp

```diff
@@ -86,6 +86,7 @@ struct ComputeExcisionBoundaryVolumeQuantities
   template <typename TargetFrame>
   using allowed_dest_tags_target_frame =
       tmpl::list<gr::Tags::SpatialMetric<3, Frame::Inertial>,
```

Comment:
We'll want `ConstraintGamma1` here as well and then handled correctly in the tpp